mne/viz/tests/test_evoked.py

```diff
@@ -168,21 +182,20 @@ def test_plot_evoked():
     evoked.plot_image(show_names=True, time_unit='s')
 
     # test groupby
-    evoked.plot_image(group_by=dict(sel=[0, 7]), axes=dict(sel=plt.axes()))
+    evoked.plot_image(group_by=dict(sel=sel), axes=dict(sel=plt.axes()))
     plt.close('all')
     for group_by, axes in (("something", dict()), (dict(), "something")):
         pytest.raises(ValueError, evoked.plot_image, group_by=group_by,
                       axes=axes)
 
-    # test plot_topo
-    evoked.plot_topo()  # should auto-find layout
-    _line_plot_onselect(0, 200, ['mag', 'grad'], evoked.info, evoked.data,
```

Comment:
why removing this one?

Reply:
There is a test_topo file, but I should maybe move it instead of delete

Reply:
I am just a bit cautious removing test in code but if you tell me it's covered else where fine.